core-feature-pack/common/src/main/resources/modules/system/layers/base/org/yaml/snakeyaml/main/module.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+Copyright 2021 Emmanuel Hugonnet (c) 2020 Red Hat, Inc..
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<module xmlns="urn:jboss:module:1.9" name="org.yaml.snakeyaml">
+    <properties>
+        <property name="jboss.api" value="private"/>
+    </properties>
+
+    <resources>
+        <artifact name="${org.yaml:snakeyaml}"/>
+    </resources>
+
+    <dependencies>
+        <module name="java.logging"/>
```

Comment:
should it be identical to https://github.com/wildfly/wildfly/blob/3eaeac0b07b4a03917f94bfb8368b6ee09c12530/ee-feature-pack/common/src/main/resources/modules/system/layers/base/org/yaml/snakeyaml/main/module.xml?

Reply:
ok, done